conjur-oss/values.schema.json

```diff
@@ -0,0 +1,234 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "required": [
+    "database",
+    "dataKey",
+    "ssl"
+  ],
+  "properties": {
+    "account": {
+      "type": "string"
+    },
+    "affinity": {
+      "type": "object"
+    },
+    "authenticators": {
+      "type": "string"
+    },
+    "conjurLabels": {
+      "type": "object"
+    },
+    "database": {
+      "required": [
+        "ssl"
+      ],
+      "properties": {
+        "password": {
+          "type": "string"
+        },
+        "ssl": {
+          "required": [
+            "key"
+          ],
+          "properties": {
+            "cert": {
+              "type": "string"
+            },
+            "key": {
+              "type": "string",
+              "minLength": 1
+            },
+            "expiration": {
+              "type": "integer",
+              "minimum": 1
+            }
+          }
+        },
+        "url": {
+          "type": "string"
+        }
+      }
+    },
+    "dataKey": {
+      "type": "string",
+      "minLength": 1
+    },
+    "deployment": {
+      "properties": {
+        "annotations": {
+          "type": "object"
+        }
+      }
+    },
+    "image": {
+      "properties": {
+        "pullPolicy": {
+          "type": "string"
+        },
+        "repository": {
+          "type": "string"
+        },
+        "tag": {
+          "type": "number"
+        }
+      }
+    },
+    "nginx": {
+      "properties": {
+        "image": {
+          "properties": {
+            "pullPolicy": {
+              "type": "string"
+            },
+            "repository": {
+              "type": "string"
+            },
+            "tag": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "nodeSelector": {
+      "type": "object"
+    },
+    "postgres": {
+      "properties": {
+        "image": {
+          "properties": {
+            "pullPolicy": {
+              "type": "string"
+            },
+            "repository": {
+              "type": "string"
+            },
+            "tag": {
+              "type": "string"
+            }
+          }
+        },
+        "persistentVolume": {
+          "properties": {
+            "create": {
+              "type": "boolean"
+            },
+            "size": {
+              "type": "string"
+            },
+            "storageClass": {
+              "type": "string"
```

Comment:
For some reason, adding this "string" type for "storageClass" causes errors when "storageClass" isn't provided. I think the fix is to comment out the "storageClass line in the values.yaml file. That is, replace:
```
    storageClass:
```
with:
```
    # Set storageClass to use a non-default storage class for your platform
    # storageClass:
```